src/VisualStudio/Setup/ProvideRoslynBindingRedirection.cs

```diff
@@ -28,17 +28,6 @@ public ProvideRoslynBindingRedirectionAttribute(string fileName)
                 PublicKeyToken = "31BF3856AD364E35",
                 OldVersionLowerBound = "0.7.0.0",
                 OldVersionUpperBound = "1.2.0.0",
-
```

Comment:
Do you want to leave a comment here for whoever merges stabilization into master?